module/item/item.mjs

```diff
@@ -1499,6 +1499,13 @@ export class SWSEItem extends Item {
             }
         };
     }
+
+    toObject() {
+        let o = super.toObject();
+        let cost = this.system.changes.filter(c => c.key === "cost");
```

Comment:
if you replace filter with find you get the first cost found rather than an array

Reply:
Fixed.